pql/pql.peg.go

```diff
@@ -287,7 +287,7 @@ func (t *tokens32) Tokens() []token32 {
 	return t.tree
 }
 
-type PQL struct {
+type pql struct {
```

Comment:
this is generated code and shouldn't be messed with... maybe we should hide it in an internal dir though